bindings/python/iota_sdk/wallet/wallet.py

```diff
@@ -51,15 +51,15 @@ def get_client(self):
         return Client(client_handle=get_client_from_wallet(self.handle))
 
     @_call_method_routine
-    def _call_method(self, name: str, data=None) -> Dict[str, str]:
+    def _call_method(self, name: str, data=None):
```

Comment:
What does this mean? It does return something, right?

Reply:
Because of the `@_call_method_routine` it returns `Any` which is the same as not specifying anything
Also this function shouldn't be used by users anyways